bagisto/bagisto
# feature : add login as customer to admin customer grid

## Issue Reference
[ #7139 ](https://github.com/bagisto/bagisto/issues/7139)

| Before | After |
| -------- | -------- |
| ![before](https://user-images.githubusercontent.com/86394952/204206565-05c11425-704c-4413-9a87-54ce9f8a0401.png) | ![after](https://user-images.githubusercontent.com/86394952/204206613-61f3816f-9461-4beb-97ac-8d1eb3b6ad36.png)


## Description
we can add a new link to the admin customer grid and after clicking on it we should be able to login as a customer.
this feature is essential because sometime admin needs to check customer panel 

## How To Test This?
1: login admin panel
2: go to manage customer 
3: in the customer grid you should be able to see a open lock icon
4: click on it and now you can login as customer

## Documentation
